src/redux/categories/categories.js

```diff
@@ -0,0 +1,14 @@
+import { CHECK_STATUS } from '../../data/constants';
+
```

Comment:
- [ **OPTIONAL** ] As I mentioned above, to correctly follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), I think it would be better if you move the concerned `categories` actions type from the `'../../data/constants'` file and come define them here by following the `ducks pattern` approach as I showed an example in the previous comments. i.e.:
```JavaScript
const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```